kata_agent.go

```diff
@@ -335,6 +410,45 @@ func (k *kataAgent) startPod(pod Pod) error {
 		hostname = hostname[:maxHostnameLen]
 	}
 
+	//
+	// Setup network interfaces and routes
+	//
+	networkNS, err := pod.storage.fetchPodNetwork(pod.id)
```

Comment:
Note: This should be part of the `pod` and we should be able to access those infos from in-memory. Opened an issue here: https://github.com/containers/virtcontainers/issues/628
Will be fixed in another PR

Reply:
ack

Reply:
done.

Reply:
You can update this cause you don't need to fetch the pod network anymore since #628 has been solved by #629